modoboa/limits/migrations/0001_squashed_0004_auto_20160413_1312.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.0.13 on 2020-01-04 17:21
```

Comment:
Why did you add this migration?

Reply:
This new migration replaces `0001_initial` to `0004_auto_20160413_1312` and was needed to prevent some unexpected issues - and headache - starting with Django 2.0. I think that this is mainly because of the `AlterModelTable` in `0002_auto_20151114_1518`. As this model changes occurred in 2016, I proceeded from the assumption that there is no more instances without them applied. Anyway, we could add a note if it is not the case to tell that such old-old-old instances should be upgraded to v1.14.0 at first.

Reply:
Ok I see. And adding a note is a good idea.